release.auto.pkrvars.hcl

```diff
@@ -8,4 +8,6 @@ source_ami_al2              = "amzn2-ami-minimal-hvm-2.0.20220912.1-x86_64-ebs"
 source_ami_al2arm           = "amzn2-ami-minimal-hvm-2.0.20220912.1-arm64-ebs"
 source_ami_al2022           = "al2022-ami-minimal-2022.0.20220831.1-kernel-5.15-x86_64"
 source_ami_al2022arm        = "al2022-ami-minimal-2022.0.20220831.1-kernel-5.15-arm64"
+kernel_version_al2022       = "-kernel-5.15"
+kernel_version_al2022arm    = "-kernel-5.15"
```

Comment:
I had to set them even though they are technically not used in the past release, because otherwise packer static-check throws 
```
Error: Unset variable "kernel_version_al2022"
A used variable must be set or have a default value
```